### PR TITLE
Support proto format

### PR DESCRIPTION
**Pull request by milarisa**

### CODE REVIEW

File: buf.gen.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+version: v1
+plugins:
+  - name: go
+    out: parser/testproto
```
Comment (sashayakovtseva) [inline buf.gen.yaml:4]: do we need this?

File: config/config.go [hunk 2]
```diff
@@ -287,6 +293,10 @@ func (cfg *Config) normallizeTask(taskCfg *TaskConfig) (err error) {
 	} else {
 		taskCfg.PromLabelsBlackList = ""
 	}
+	if taskCfg.Parser == "proto" && cfg.SchemaRegistry.URL == "" {
+		err = errors.Newf("For parser %s schema registry needs", taskCfg.Parser)
```
Comment (sashayakovtseva) [inline config/config.go:297]: maybe "Schema registry is required for parser %s"?

File: task/task.go [hunk 3]
```diff
@@ -17,6 +17,7 @@ package task
 
 import (
 	"fmt"
+	"github.com/confluentinc/confluent-kafka-go/schemaregistry"
```
Comment (sashayakovtseva) [inline task/task.go:20]: nit: how about separating std and 3s party imports?

File: parser/proto.go [hunk 4]
```diff
@@ -0,0 +1,471 @@
...
+//go:generate mockgen -source=../vendor/github.com/confluentinc/confluent-kafka-go/schemaregistry/schemaregistry_client.go -imports=schemaregistry=github.com/confluentinc/confluent-kafka-go/schemaregistry -package=parser -mock_names=Client=MockSchemaRegistryClient -destination=schema_registry_mock_test.go
+
+// ProtoParser knows how to get data from proto format.
+type ProtoParser struct {
+	pp           *Pool
+	deserializer *ProtoDeserializer
+}
+
+// Parse parses bytes into metric that knows how.
+func (p *ProtoParser) Parse(bs []byte) (metric model.Metric, err error) {
```
Comment (sashayakovtseva) [inline parser/proto.go:35]: how about getting rid of named returned values? here and elsewhere in this file

**Review by sashayakovtseva — APPROVED** [review]

File: buf.gen.yaml (hunk 1 above)
Comment (milarisa) [reply buf.gen.yaml:4]: It's necessary for proto parser unit testing. I haven't any ideas how do it more correctly.

File: parser/proto.go (hunk 4 above)
Comment (milarisa) [reply parser/proto.go]: Yes, fixed this file

File: config/config.go (hunk 2 above)
Comment (milarisa) [reply config/config.go]: I agree, fixed

File: task/task.go (hunk 3 above)
Comment (milarisa) [reply task/task.go]: Fixed

**Review by proxeter — APPROVED** [review]

**Pull request merged**